spec/index.bs

```diff
@@ -11268,32 +11272,37 @@ None of the depth formats can be filtered.
         <td>1
         <td>{{GPUTextureSampleType/"uint"}}
         <td colspan=2>&checkmark;
+        <td>{{GPUTextureFormat/stencil8}}
     <tr>
         <td>{{GPUTextureFormat/depth16unorm}}
         <td>2
         <td>depth
         <td>2
         <td>{{GPUTextureSampleType/"depth"}}
         <td colspan=2>&checkmark;
+        <td>{{GPUTextureFormat/depth16unorm}}
     <tr>
         <td>{{GPUTextureFormat/depth24plus}}
         <td>4
         <td>depth
         <td>- (<a href="#depthPlus">See prose</a>)
         <td>{{GPUTextureSampleType/"depth"}}
         <td colspan=2>&cross;
+        <td>- (Not a valid image copy source nor a valid image copy destination)
```

Comment:
nit: I think for the PR that @austinEng is doing we might need these aspect-specific formats listed even if they are not copyable. But feel free to defer to that PR if you'd like.